### PR TITLE
events: fix notification methods for image expiry events (PROJQUAY-7487)

### PR DESCRIPTION
**Pull request by Sunandadadi**
Passing notification uuid to fetch matching notification event.

### CODE REVIEW

File: data/model/notification.py [hunk 1]
```diff
@@ -263,6 +265,11 @@ def list_repo_notifications(namespace_name, repository_name, event_name=None):
             .where(ExternalNotificationEvent.name == event_name)
         )
 
+    if notification_uuid:
+        query = query.switch(RepositoryNotification).where(
```
Comment (bcaton85) [inline data/model/notification.py:269]: i don't think you need this switch since `RepositoryNotification` is the table being selected from

Comment (Sunandadadi) [reply data/model/notification.py]: yes, fixed in the latest commit.

**Review by bcaton85 — APPROVED** [review]
LGTM

**Review by kleesc — APPROVED** [review]

**Pull request merged**